Resources/Private/BackendFusion/Components/FlashMessages.fusion

```diff
@@ -0,0 +1,16 @@
+prototype(Neos.GoogleAnalytics:Component.FlashMessages) < prototype(Neos.Fusion:Component) {
+    flashMessages = ${[]}
+
+    renderer = afx`
+        <div id="neos-notification-container" class="neos-notification-top" @if.hasMessages={props.flashMessages}>
+            <Neos.Fusion:Loop items={props.flashMessages} itemName="message">
+                <div class="neos-notification neos-notification-error">
+                    <div class="neos-notification-content">
+                        <i class="fas fa-error"></i>
+                        <div class="neos-notification-heading" @if.hasTitle={message.title}>{message.message}</div>
```

Comment:
The title never gets shown. Is that on purpose?

Reply:
No that was a mistake, I updated the prototype and split the rendering. Please check.